### PR TITLE
EdPlatform rework + project clean

### PR DESCRIPTION
**Pull request by DanyloNaumenko**
To many changes, no comments.

### CODE REVIEW

File: Managers/EdPlatformManager.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        public bool RemoveStudent(Student student)
+        {
+            return EdPlatformInstance.Students.RemoveStudent(student.Id);
+        }
+        public bool RemoveTutor(Tutor tutor)
+        {
+            return EdPlatformInstance.Tutors.RemoveTutor(tutor.Id);
+        }
+
+        public void GetSupportInfo()
```
Comment (Vadim211) [inline Managers/EdPlatformManager.cs:35]: This can be record type as it was required in prev home work

File: Program.cs [hunk 2]
```diff
@@ -20,28 +37,25 @@ static void Main()
             studentList.AddStudent(1234573, "2016", "Manuel", "Pérez");
             studentList.AddStudent(1234574, "2017", "Isabel", "Sánchez");
             studentList.AddStudent(1234575, "2016", "Francisco", "Ramírez");
```
Comment (Vadim211) [inline Program.cs]: I suppose comments can be deleted

**Review by Vadim211 — CHANGES_REQUESTED** [review]

File: Extentions/LessonExtension.cs [hunk 3]
```diff
@@ -16,14 +17,14 @@ public static void StudentPresent(this Lesson lesson, int id)
             {
                 if (student.Id == id)
```
Comment (IlliaKrasheninnikov) [inline Extentions/LessonExtension.cs:18]: Student ID was changed to Guid id

File: Managers/OutputManager.cs
Comment (IlliaKrasheninnikov) [inline Managers/OutputManager.cs:1]: Can you please add Input manager. Just copy from Program.cs UserInput function.

**Review by VikMoskalenko — APPROVED** [review]

File: Managers/OutputManager.cs
Comment (DanyloNaumenko) [reply Managers/OutputManager.cs:1]: Done

**Pull request merged**